launch/bagfile_vicon.launch

```diff
@@ -5,13 +5,15 @@
     <param name="vicon_estimator/dt" value="0.01" />
     <param name="translational_estimator/kp" value="1.0" />
     <param name="translational_estimator/kv" value="0.1" />
+    <param name="translational_estimator/outlier_threshold_m_" value="0.5" />
+    <param name="translational_estimator/maximum_outlier_count_translation_" value="30" />
     <param name="rotational_estimator/orientation_estimate_initial_covariance" value="1" />
     <param name="rotational_estimator/rate_estimate_initial_covariance" value="1" />
     <param name="rotational_estimator/orientation_process_covariance" value="0.00001" />
     <param name="rotational_estimator/rate_process_covariance" value="10" />
     <param name="rotational_estimator/orientation_measurementCovariance" value="0.005" />
     <param name="rotational_estimator/outlier_threshold_degrees" value="30" />
-    <param name="rotational_estimator/maximum_outlier_count" value="50" />
+    <param name="rotational_estimator/maximum_outlier_count_rotation_" value="30" />
```

Comment:
Same. Also note, no trailing underscore (_) for parameter names. This is only for member variables.